samples/matter/common/src/dfu/smp/dfu_over_smp.cpp

```diff
@@ -130,4 +158,24 @@ void DFUOverSMP::StartServer()
 	ChipLogProgress(DeviceLayer, "DFU over SMP started");
 }
 
+void DFUOverSMP::Disconnected(bt_conn *conn, uint8_t reason)
+{
+	if (!sDfuInProgress) {
+		return;
+	}
+
+	bt_conn_info btInfo;
+
+	bt_conn_get_info(conn, &btInfo);
```

Comment:
This function can return an error on failure, it's worth to consider such a case.

Reply:
Done